src/com/magento/idea/magento2plugin/actions/generation/generator/util/eav/ProductAttributeMapper.java

```diff
@@ -71,6 +71,11 @@
         mappedAttributes.put(EavAttribute.VISIBLE_ON_FRONT.getAttribute(),
                 Boolean.toString(eavEntityData.isVisibleOnFront()));
 
+        if (eavEntityData.getApplyTo() != null && !eavEntityData.getApplyTo().isEmpty()) {
+            mappedAttributes.put(EavAttribute.APPLY_TO.getAttribute(),
+                    wrapStringValueForTemplate(eavEntityData.getApplyTo()));
```

Comment:
@ProkopovVitaliy, just want to bring your attention, that breaking a multiple arguments to a new line, should move all the arguments to a new empty line, like following.

No action is needed here, just wanted to add a remark, it also applies to PHP code.
```suggestion
            mappedAttributes.put(
                    EavAttribute.VISIBLE_ON_FRONT.getAttribute(),
                    Boolean.toString(eavEntityData.isVisibleOnFront())
            );
```

Reply:
@eduard13 Thanks for recommendation, really it's looks better in your variant. I will take this into account.

Thanks!